README.md

```diff
@@ -92,7 +92,7 @@ This project was developed and is maintained by HMS Networks, using the Ewon Jav
 
 **Step 1:** Connect your computer to the LAN of your Ewon device, then open an FTP connection to your Ewon device.
 
->To create an FTP connection, using your file manager (i.e. Windows Explorer), enter `ftp://[EWON IP HERE]/` in the address bar, and hit enter.
+>To create an FTP connection, using your file manager (i.e. Windows Explorer -- Not your web browser), enter `ftp://[EWON IP HERE]/` in the address bar, and hit enter.
```

Comment:
Windows explorer was renamed to file explorer in Windows 10. I am still unsure about adding -- Not your web browser as  I do think its a little harsh/rude.

Reply:
changed to File Explorer and removed the web browser part.